tensorflow/python/ops/linalg_ops.py

```diff
@@ -403,12 +404,13 @@ def svd(tensor, full_matrices=False, compute_uv=True, name=None):
       `[..., M, M]`. Not returned if `compute_uv` is `False`.
     v: Right singular vectors. If `full_matrices` is `False` (default) then
       shape is `[..., N, P]`. If `full_matrices` is `True` then shape is
-      `[..., N, N]`. Not returned if `compute_uv` is `False`.
+      `[..., N, N]`. Not returned if `compute_uv` is `False`. 
 
   @compatibility(numpy)
-  Mostly equivalent to numpy.linalg.svd, except that the order of output
-  arguments here is `s`, `u`, `v` when `compute_uv` is `True`, as opposed to
-  `u`, `s`, `v` for numpy.linalg.svd.
+  Mostly equivalent to numpy.linalg.svd, except that the transposition and 
```

Comment:
"transposition" of outputs does not make sense. How about this:

Mostly equivalent to numpy.linalg.svd, except that order of output arguments here is `s`, `u`, `v` when `compute_uv` is `True`, as opposed to `u`, `s`, `v` for numpy.linalg.svd. Also notice that this function returns `v` as opposed to `transpose(conj(v))`  for numpy.linalg.svd, which does not follow the mathematical definition of the SVD.

Reply:
Maybe say "numpy.linalg.svd returns v" instead of "this function returns v" to be explicit?

@rmlarsen agree on numpy choosing suboptimal output format

Reply:
@yaroslavvb I agree that your formulation is clearer. Thanks.

Reply:
ugh actually, I think it should be tf.linalg.svd return v. numpy.linalg.svd returns v^H, but calls the argument "v"... :-)